src/components/MultipleAvatars.tsx

```diff
@@ -157,6 +157,7 @@ function MultipleAvatars({
                         name={icons[0].name}
                         type={icons[0].type}
                         fallbackIcon={icons[0].fallbackIcon}
+                        displayInDefaultIconColor={icons[0].type !== CONST.ICON_TYPE_WORKSPACE}
```

Comment:
Shouldn't we add the same in https://github.com/Expensify/App/blob/01f652b538415e3ae8418ad8015d00cb92370625/src/components/MultipleAvatars.tsx#L275?